packages/python-importlib-resources/python-importlib-resources.spec

```diff
@@ -0,0 +1,49 @@
+# Created by pyp2rpm-3.3.3
+%global pypi_name importlib-resources
+
+Name:           python-%{pypi_name}
+Version:        5.0.0
```

Comment:
packaging 5.0.0 instead of the newer releases, as 5.0.2 introduced a dependency on "find_namespaces" from setuptools that is not available in our setuptools 39 (and I really would prefer not to fiddle around with setuptools)